Mono.Addins.Setup/Mono.Addins.Setup/Repository.cs

```diff
@@ -37,7 +37,7 @@
 
 namespace Mono.Addins.Setup
 {
-	internal class Repository
+	public class Repository
```

Comment:
Ok, but please make internal everything that is not needed by marketplace integration